schemas/metadata/structured-ingestion/structured-ingestion.1.schema.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
It's not clear to me where best to put this schema. The "metadata" namespace seemed potentially least confusing and I'm assuming that the other schemas in that folder aren't intended to be tables in BQ (which is the case for this schema as well).

Reply:
assumption confirmed - the other schemas in here are used for stuff in the metadata bucket (`sources.json` which is used by the Dataset API) and credentials files which are used by a few applications to connect to databases.

Reply:
Would people or tooling freak out if I created a `templates/metadata/README.md` to document the intention of the schemas in this namespace?

Reply:
I don't think so

Reply:
:mdboom put a changelog into the glean namespace. That could also be a useful feature here as we change/add metadata in the future. And +1 on the readme.

Reply:
The structure here is complete and should be matching the implementation in https://github.com/mozilla/gcp-ingestion/pull/542

I'll take this out of draft mode on Monday after I add the README/changelog.

Reply:
Added.